python/Ice/greeter/README.md

```diff
@@ -0,0 +1,31 @@
+# Greeter
+
+The Greeter demo illustrates how to send a request and wait for the response.
+
+This demo provides two implementations for the server: a synchronous dispatch implementation (in `server`), and an asynchronous dispatch implementation (in `server_amd`). The same client works with both.
+
+We recommend running each application in a separate Python virtual environment. If you are new to Python virtual environments, see [Python Virtual Environments].
+
+## Running the server
+
+Create and activate a Python virtual environment in the `server` or `server_amd` directory to run the synchronous or asynchronous version of the server application, respectively.
+
+From the activated virtual environment in the `server` or `server_amd` directory, install the program dependencies and run the application using the following command:
+
+```shell
+pip install -r requirements.txt
+python3 main.py
```

Comment:
I think should be consistent with usage of `pip/python` and `pip3/python3`. The python docs seem to prefer to just use `pip` and `python` these days.

Reply:
not sure if all platforms provide python alias...

Reply:
They do inside the virtualenv. If you look at the doc page you link they use `python` exclusively